docs/_static/css/theme_custom.css

```diff
@@ -96,35 +96,49 @@ input[type="file"]:focus:invalid:focus,input[type="radio"]:focus:invalid:focus,i
 
 code,.rst-content tt,.rst-content code {
    color: #E6A160;
+   border: #717577;
 }
 
-.wy-menu-vertical li.on a,.wy-menu-vertical li.current>a {
-   background: #333;
+/* first level headers */
+.wy-menu-vertical li.current > a, .wy-menu-vertical li.current > a:hover {
+  color: #e6a150;
+  background: #333;
+}
+
+/* second and third level nested headers */
+.wy-menu-vertical li.current li.current > a, .wy-menu-vertical li.current li.current a {
+  color: #413a33;
+  background: #c9c9c9;
 }
 
-.wy-menu-vertical li.on a:hover,.wy-menu-vertical li.current>a:hover {
-   background: #333
+/* second and third level nested headers */
+.wy-menu-vertical li.current li.current > a:hover, .wy-menu-vertical li.current li.current a:hover {
+  background: #aeaeae;
 }
 
 .wy-menu-vertical a:active {
-   background-color: #E6A150;
+  background-color: #E6A150;
 }
 
 .wy-side-nav-search {
-   background-color: #E6A150;
-   color: #333;
+   background-color: #333333;
+   color: #e6a150;
 }
 
 .wy-side-nav-search img {
-   background-color: #E6A150;
+   background-color: #e6a150;
 }
 
 .wy-side-nav-search>a,.wy-side-nav-search .wy-dropdown>a {
    color: #333;
 }
 
 .wy-side-nav-search>div.version {
-   color: #666;
+   color: #bbb;
+}
+
+.wy-side-nav-search input[type=text] {
+    background-color: #e3e3e3;
```

Comment:
Yeah, this is a good catch. I tried to get rid of FFF backgrounds everywhere else but missed this one.